### PR TITLE
profile

### PR DESCRIPTION
**Pull request by quangchien0806**

### CODE REVIEW

**Comment by quangchien0806** [comment]
[add Profile Section](https://trello.com/c/nX62hfLO/61-add-profile-section)

File: src/components/CardFile/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  </Menu>
+);
+
+const CardFile = () => {
+  return (
+    <div className="card__body card__file">
+      {dataFile.map((data, idx) => (
+        <div className="card__file-item" key={idx}>
+          {imgExt.includes(data.title.split(".").pop()) ? (
+            <div className="card__file-img">
```
Comment (phongnguyenhh1996) [inline src/components/CardFile/index.js:22]: check file type here

File: src/components/CardFile/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { RiDownload2Line } from "react-icons/ri";
+import { dataFile } from "./dataFile";
+import "./style.scss";
+import Menu, { Item as MenuItem, Divider } from "rc-menu";
+
+const imgExt = ["jpg", "svg", "png", "webp"];
+
+const menu = ["Action", "Another Action", "Divider", "Delete"];
+
+const dropDownMenu = (
```
Comment (phongnguyenhh1996) [inline src/components/CardFile/index.js]: create component DropdownMenu

File: src/components/User/index.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import {
+  BsPencilFill,
+  BsFillRecordCircleFill,
+  BsChevronDown,
+} from "react-icons/bs";
+import Dropdown from "rc-dropdown";
+import "rc-dropdown/assets/index.css";
```
Comment (phongnguyenhh1996) [inline src/components/User/index.js]: move to app.js

File: src/components/User/index.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+import "rc-menu/assets/index.css";
+import Menu, { Item as MenuItem } from "rc-menu";
+import "./style.scss";
+
+const act = ["Available", "Busy"];
+
+const dropDownAct = (
```
Comment (phongnguyenhh1996) [inline src/components/User/index.js]: using DropdownMenu component

File: src/components/profile/index.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { FiMoreVertical } from "react-icons/fi";
+import { BiChevronRight } from "react-icons/bi";
+import User from "../User";
+import Avatar from "../../asset/img/avatar.jpg";
+import Dropdown from "rc-dropdown";
+import Menu, { Item as MenuItem, Divider } from "rc-menu";
+import "rc-dropdown/assets/index.css";
```
Comment (phongnguyenhh1996) [inline src/components/profile/index.js]: move to app.js

File: src/components/profile/index.js [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+import "rc-collapse/assets/index.css";
+import Collapse, { Panel } from "rc-collapse";
+import { AiOutlineUser } from "react-icons/ai";
+import { MdAttachFile } from "react-icons/md";
+import CardBody from "../CardBody";
+import CardFile from "../CardFile";
+
+const menu = ["Edit", "Action", "Divider", "Another action"];
+
+const dropDownMenu = (
```
Comment (phongnguyenhh1996) [inline src/components/profile/index.js]: DropdownMenu component

File: src/components/profile/index.js [hunk 7]
```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { FiMoreVertical } from "react-icons/fi";
+import { BiChevronRight } from "react-icons/bi";
+import User from "../User";
+import Avatar from "../../asset/img/avatar.jpg";
```
Comment (phongnguyenhh1996) [inline src/components/profile/index.js]: rename to avatarImg

**Review by phongnguyenhh1996 — APPROVED** [review]

**Pull request merged**